### ISSUE TITLE
UX improvement: Clearer visualization LaTeX-compiler running / success / error, add kill button

### ISSUE THREAD

**Issue by p10n** [James-Yu/Atom-LaTeX#197]
This is critics on a high level, just from my personal experience.

The gray log messages at the the bottom are nice, but one easily overlooks when the compiler is actually running. I suggest the following:

The "build LaTeX" button (with the PLAYBACK icon) should turn into a STOP button with a spinning icon, while the compiler is running. That would clearly indicate whats happening, and provide a easy-to-find means to kill the compiler when accidentally started. (This is especially useful for new users who are not familiar with the compile-on-save feature. I happend to modify a few included files shortly one after the other, and I think I was triggering multiple compiler instances causing mysterious compile errors. A clearly visible stop button would have cleared that situation.)

When the document was built successfully, the button should turn green - or maybe the "successfully built" line should be green.

The gray log messages are nice and subtle, on the other hand the red error boxes (and blue info boxes) are quite big. But success is so subtle. I think there should be a nice indication of success. Dont we all like positive news? :-)

**Comment by ashthespy** [comment]
>The "build LaTeX" button (with the PLAYBACK icon) should turn into a STOP button with a spinning icon, while the compiler is running. That would clearly indicate whats happening, and provide a easy-to-find means to kill the compiler when accidentally started. (This is especially useful for new users who are not familiar with the compile-on-save feature. I happend to modify a few included files shortly one after the other, and I think I was triggering multiple compiler instances causing mysterious compile errors. A clearly visible stop button would have cleared that situation.)

>But success is so subtle. I think there should be a nice indication of success. Dont we all like positive news? :-)

There is a already status icon, but it's tucked away into the status bar at the left end. It shows the compilation status and failures/warning/success status icons. 

Good idea about making the play turn into a stop or kill button while compiling.

**Comment by mmarras** [comment]
Yes being able to kill would be nice. Also would be nice to have an autokill feature toggle-able in the settings, that as soon as you hit compile another time, the old process is killed. Occasionally, I find some stray pdftex processes running that never finish compiling because of syntax problems.

**Comment by ashthespy** [comment]
@mmarras  by design it should [kill](https://github.com/ashthespy/Atom-LaTeX/blob/master/lib/builder.coffee#L15) previous process when launching a new build. Else, the kill build command is currently hidden behind a keyboard/menu command..
